Business Social Network/AppModules/Search/views/searchResults.html

```diff
@@ -1,66 +1,65 @@
-<html>
-
-<div class = "container"
-  ng-controller = "SearchBoxController"
+<div class = "container" ng-controller = "SearchResultsController"
   ng-show = "searchTerm.length > 0">
 
+<h1 ng-if = "searchResults.length != 1">
+  Your search for "{{searchTerm}}" returned {{searchResults.length}} results
```

Comment:
@Swailem95 
please user ng-bind instead of {{}} as this will not always work 
http://stackoverflow.com/a/16126174